typescript/sdk/src/gas/HyperlaneIgpDeployer.ts

```diff
@@ -111,14 +114,25 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
         !actual.gasPrice.eq(desired.gasPrice) ||
         !actual.tokenExchangeRate.eq(desired.tokenExchangeRate)
       ) {
-        this.logger.debug(
-          `-> ${remote} ${serializeDifference(actual, desired)}`,
+        this.logger.info(
+          `-> ${remote} - ${serializeDifference(actual, desired)}`,
```

Comment:
```suggestion
          `-> ${remote}: ${serializeDifference(actual, desired)}`,
```
maybe colon instead to avoid any possible confusion of +ve/-ve diffs